src/parameterizations/vertical/MOM_CVMix_KPP.F90

```diff
@@ -145,11 +150,15 @@ module MOM_CVMix_KPP
   integer :: id_EnhW     = -1
   integer :: id_La_SL    = -1
   integer :: id_OBLdepth_original = -1
+  integer :: id_StokesXI = -1
+  integer :: id_Lam2     = -1
   !>@}
 
   ! Diagnostics arrays
-  real, allocatable, dimension(:,:)   :: OBLdepth  !< Depth (positive) of ocean boundary layer (OBL) [Z ~> m]
+  real, pointer,     dimension(:,:)   :: OBLdepth  !< Depth (positive) of ocean boundary layer (OBL) [Z ~> m]
   real, allocatable, dimension(:,:)   :: OBLdepth_original  !< Depth (positive) of OBL [Z ~> m] without smoothing
+  real, allocatable, dimension(:,:)   :: StokesParXI !< Stokes similarity parameter
```

Comment:
Please add a description of the units of these two new real variables in the comments describing them.